app/models/feedback_reporters/feedback_reporter.rb

```diff
@@ -0,0 +1,37 @@
+class FeedbackReporter
+
+  attr_accessor :title, 
+    :description, 
+    :category, 
+    :email
+
+  def initialize(attrs={})
+    attrs.each_pair do |key, val|
+      self.send("#{key}=", val)
+    end
+  end
+
+  def send_report!
+    HTTParty.post("#{ArchiveConfig.BUGS_SITE}/projects/#{project_id}/bugs",
+      headers: { 
```

Comment:
Align the parameters of a method call if they span more than one line.<br>Trailing whitespace detected.